src/components/DotaMap/DotaMap.css

```diff
@@ -0,0 +1,23 @@
+.map {
```

Comment:
I think we should just use detailed only, unless we have a specific need to use both map images.

Also the Dota 2 wiki has links to multiple map versions (pre 6.86, pre 6.82 etc.)  perhaps we should add those too?